.dockerignore

```diff
@@ -0,0 +1,9 @@
+.git
+.gitignore
+.vscode
+node_modules
+npm-debug.log
+Dockerfile*
+docker-compose*
+README.md
+LICENSE
```

Comment:
Should this have .md extension?

Reply:
Indeed, I'll add a wild card for markdown files.

Reply:
Cool. Just making sure :-)